Kernel/Prekernel/Arch/aarch64/GPIO.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021, Nico Weber <thakis@chromium.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Array.h>
+#include <AK/Types.h>
+
+namespace Prekernel {
+
+struct GPIOControlRegisters;
+
+// Can configure the general-purpose I/O registers on an Raspberry Pi.
```

Comment:
```suggestion
// Can configure the general-purpose I/O registers on a Raspberry Pi.
```

Reply:
(I had "RPi" first, but then forgot to update the article.)